misc/encoding/src/index.ts

```diff
@@ -155,6 +153,7 @@ export function binaryToNumber(bin: string): number {
 // -- Validators ----------------------------------------- //
 
 export function isBinaryString(str: any): boolean {
+  // eslint-disable-next-line prefer-regex-literals
```

Comment:
Wouldn't we not need this ignore anymore with the new lint rules

Reply:
Good catch thank you 👍